vidarrtest-regression.json.in

```diff
@@ -423,7 +432,7 @@
             {
                 "metrics_calculate": "@CHECKOUT@/tests/calculate.sh",
                 "metrics_compare": "@CHECKOUT@/tests/compare.sh",
-                "output_metrics": "/.mounts/labs/gsi/testdata/mutect/output_metrics/2.0/MutectExtraArgumentsTest.metrics",
```

Comment:
Again, I would avoid changing test names which do not contain any sample-identifying data. The names of the metric files are informing us on the test setup and should be left as they were